src/sdk/model/edge.ts

```diff
@@ -144,6 +158,217 @@ export class Edge extends Entity {
     });
   }
 
+  /**
+   * Get the Edge dhcp pool
+   * @returns {Promise<EdgeDhcpJson>}
+   */
+  async getDhcp(): Promise<EdgeDhcpJson> {
```

Comment:
Why are you not wrapping all of the *Json interfaces with corresponding classes like we've done with everything else in this SDK?

Reply:
I don't understand what you're talking about :-/.

Reply:
Ho you mean i should create a ts class instead of only getting the Json interface ? Because a class is not needed and not even usefull in our case. We don't need to instanciate a class. We only need to get the value from the API. If we needed to do object manipulation we would have create a class but there, it's not needed. When we would add the `POST`, `PUT` and `DELETE` parts, we would create a builder class.

Reply:
I wouldn't say that its not useful. Some good reasons to continue doing this are: it allows us to provide convenience methods associated with the data (e.g. we could add a method to get an array of all IPs on an edge inteface), to convert Json types to more useful javascript types (e.g. to convert a number representing epoch millis from the API to a javascript date), and to provide a buffer between server response format and the client integration code so that we can have better control over when we need to break compatibility with future changes. I would definitely like to continue with this pattern that we've been using thus far.

Reply:
ok i see you point. Well. I'll create classes for all of this JSON object though.